docs/_locale/de_DE/LC_MESSAGES/api.po

```diff
@@ -480,7 +480,7 @@ msgid "Current value of the 'Content-Length' header."
 msgstr ""
 
 #: ../../../bottle.pydocstring of bottle.FileUpload.get_header:1
-msgid "Return the value of a header within the mulripart part."
+msgid "Return the value of a header within the multipart part."
```

Comment:
My logic for updating translations is that the translator has either picked a translation or hasn't gotten around to it, and in either case fixing a typo isn't material.

If there's a sign that the translation was misguided based on the typo, then I wouldn't update a key.